pkg/scheduler/queue/queue.go

```diff
@@ -655,38 +659,46 @@ func FirstTenant() TenantIndex {
 	return TenantIndex{last: -1}
 }
 
-// waitingQuerierConn is a "request" indicating that the querier is ready to receive the next query request.
-// It embeds the unbuffered `recvChan` to receive the requestForQuerier "response" from the RequestQueue.
+// QuerierWorkerDequeueRequest is a request from a querier-worker which is ready to receive the next query.
+// It embeds the unbuffered `recvChan` to receive the querierWorkerDequeueResponse "response" from the RequestQueue.
 // The request/response terminology is avoided in naming to disambiguate with the actual query requests.
```

Comment:
This comment may be out of date? Since the types are renamed to include request / response